modules/core/src/viewports/globe-viewport.ts

```diff
@@ -59,37 +62,39 @@ export type GlobeViewportOptions = {
 };
 
 export default class GlobeViewport extends Viewport {
-  // @ts-ignore
-  longitude: number;
-  // @ts-ignore
-  latitude: number;
-  resolution: number;
+  longitude!: number;
+  latitude!: number;
+  resolution!: number;
 
   constructor(opts: GlobeViewportOptions = {}) {
     const {
       latitude = 0,
       longitude = 0,
       zoom = 0,
-      nearZMultiplier = 0.1,
-      farZMultiplier = 2,
+      nearZMultiplier = 0.5,
+      farZMultiplier = 1,
       resolution = 10
     } = opts;
 
-    let {height, altitude = 1.5} = opts;
+    let {height, altitude = 1.5, fovy} = opts;
 
     height = height || 1;
-    altitude = Math.max(0.75, altitude);
+    if (fovy) {
+      altitude = fovyToAltitude(fovy);
+    } else {
+      fovy = altitudeToFovy(altitude);
+    }
+    // Used to match globe and web mercator projection at high zoom
+    const scaleAdjust = 1 / Math.PI / Math.cos((latitude * Math.PI) / 180);
```

Comment:
Is this the new math logic? Almost missed it. Not sure we can put this in a separate method or something to make it stand out mode?
Commenting generously here would be appreciated.